pkg/test-infra/tidb/operation.go

```diff
@@ -109,12 +109,10 @@ func (o *Ops) GetNodes() ([]clusterTypes.Node, error) {
 func (o *Ops) GetClientNodes() ([]clusterTypes.ClientNode, error) {
 	var clientNodes []clusterTypes.ClientNode
 
-	k8sNodes, err := o.getK8sNodes()
+	ips, err := util.GetNodeIPs(o.cli, o.ns, map[string]string{"app.kubernetes.io/instance": o.name})
```

Comment:
The reason we use getK8sNodes is that we want local machine can connect to the TiDB service deployed on K8s cluster, it's convenient when deployment. If you change to retrieve a pod or a clusterIp, we lost this conveniency.

Reply:
1. previous `o.getK8sNodes` may get a rand (but may wrong) machine IP with multiple k8s nodes.
2. I think this `util.GetNodeIPs` can get local machine IP

Reply:
That's OK.